GitCommands/Git/GitModule.cs

```diff
@@ -2198,7 +2198,7 @@ IReadOnlyList<Remote> ParseRemotes(IEnumerable<string> lines)
                         var fetchLine = enumerator.Current;
 
                         // An invalid module is not an error; we simply return an empty list of remotes
-                        if (fetchLine.Contains("not a git repository"))
+                        if (fetchLine.IndexOf("not a git repository", StringComparison.OrdinalIgnoreCase) >= 0)
```

Comment:
Why not keeping Contains but with `Contains("not a git repository", StringComparison.OrdinalIgnoreCase)`?

Reply:
Because that is an extension method
![image](https://user-images.githubusercontent.com/4403806/57854118-c1639800-782a-11e9-955a-a25962ea1fec.png)

And in my past experience it sometimes causes troubles...